docs/pages/web/tiledata.js

```diff
@@ -0,0 +1,185 @@
+              />
+            }
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <AccessibilitySection
+        name={generatedDocGen?.displayName}
+        description="Users should be able to navigate or activate TileData using a keyboard or other input modalities. Be sure to include an `accessibilityLabel` for the screen reader if you're using the `trend` or `tooltip` props."
+      />
+
+      <MainSection
+        name="Localization"
+        description={`
+        Be sure to localize \`title\`, \`value\`, \`trend.accessibilityLabel\`, and \`tooltip.accessibilityLabel\` in TileData. 
+
+        When the title of the TileData reaches its max width, either intentionally or through localization, the title will wrap as needed to display the full text. Keep this in mind when selecting wording for your TileData menu items. Note that localization can lengthen text by 20 to 30 percent. `}
+      />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          description="TileData can be used along side the colors provided from the Data Visualization [Color Palette](https://gestalt.pinterest.systems/foundations/data_visualization/palette#12-Color-categorical-palette). You may use colors to distinguish different data lines."
+          title="Colors"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={color} name="Colors Variant" />}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description="Use `tooltip` to display clarifying information on hover/focus. We recommend using tooltips when trying to provide the user additional context/details."
+          title="Tooltip"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={tooltip} name="Tooltip variant" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          description="Disabled TileData cannot be interacted with using the mouse or keyboard. This is commonly used to disable interaction when there are pending permissions or data pre-requisites have not been met."
+          title="Disabled"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={disabled} name="Disabled variant" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          description={`Use checkboxes when enabling a multi-select experience. You can show a checkbox state by passing the \`showCheckbox\` prop.`}
+          title="Group"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={group} name="Checkbox Variant" />}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <QualityChecklist component={generatedDocGen?.displayName} />
+
+      <MainSection name="Related">
+        <MainSection.Subsection
+          description={`
+      **[Datapoint](/web/Datapoint)**
```

Comment:
This is still incorrect — URLs are case-sensitive, and all our component paths are completely lowercase

Reply:
oops, got it! Fixed